ansible/ansible.cfg

```diff
@@ -8,6 +8,7 @@ remote_user         = root
 retry_files_enabled = False
 roles_path          = roles
 squash_actions      = apk
+deprecation_warnings=False
```

Comment:
Should there be a space after the =

Reply:
Shouldn't the deprecated entries be updated instead? I understand it may not be doable for this PR, but I'd rather live with the noise than add this line here. I have updated scripts because of deprecation warnings before, this might give us a bit of forward warning.

Reply:
removed